components/layout/construct.rs

```diff
@@ -198,8 +198,17 @@ impl InlineFragmentsAccumulator {
 
         match enclosing_style {
             Some(enclosing_style) => {
```

Comment:
nit: While you're here, can you change this to `if let Some(enclosing_style) = enclosing_style` and reduce indentation in the code below?